lib/sandbox.js

```diff
@@ -426,7 +431,7 @@ Sandbox.prototype.updateWebtask = function (options, cb) {
             if (typeof options.parseBody !== 'undefined' || typeof options.parse !== 'undefined' || claims.pb) {
                 newClaims.pb = +(options.parseBody || options.parse || claims.pb);
             }
-            if (typeof options.mergeBody !== 'undefined' || typeof options.merge !== 'undefined' || claims.pb) {
+            if (typeof options.mergeBody !== 'undefined' || typeof options.merge !== 'undefined' || claims.mb) {
```

Comment:
Wow, that one certainly found a spot between the cracks for a while. Nice catch.